code/game/machinery/camera/camera.dm

```diff
@@ -129,6 +129,18 @@
 	if(!health && status)
 		deactivate(user, 0)
 
+/obj/machinery/camera/attack_animal(mob/living/simple_animal/user)
+	if(!istype(user))
+		return
+	if(user.melee_damage_upper >= 10)
```

Comment:
kor, why. generate the damage from upper and lower, then check if it's above 10 to see if you damage the camera(and insert appropriate feedback if it fails to do enough damage)